config.go

```diff
@@ -833,8 +834,8 @@ func loadConfig() (*config, []string, error) {
 
 		if cfg.TorIsolation &&
 			(cfg.ProxyUser != "" || cfg.ProxyPass != "") {
-			btcdLog.Warn("Tor isolation set -- overriding " +
-				"specified proxy user credentials")
+			fmt.Fprintln(os.Stderr, "Tor isolation set -- "+
```

Comment:
That is correct, however, people tend to copy paste and now by having everything identical there is no excuse to do it wrong.  Larting can then ensue appropriately.